### DISCUSSION TITLE
How can I disable `npm` plugin?

### DISCUSSION THREAD

**Discussion by arijitAD** [semantic-release/semantic-release#1883] [Configuration] [answered]
I am using `semantic-release` for my Golang project and I don't want to provide `npm` keys.

**Comment by mhagnumdw** [comment, marked as answer]
@arijitAD , if you define the list of plugins and do not specify `@semantic-release/npm`, it will be disabled. I tested and it worked.

eg: https://semantic-release.gitbook.io/semantic-release/support/faq#can-i-use-semantic-release-to-publish-non-javascript-packages

**Comment by kddsultan** [comment]
I set the package.json `"private": "true"` and set empty '' string as token.
